### PR TITLE
export poll results in motion pdfs

### PR DESCRIPTION
**Pull request by MaximilianKrambach**
mostly copied from 2.3 client; also a bit of cleanup of old documentation and unused functions on the way.

### CODE REVIEW

**Comment by MaximilianKrambach** [comment]
> Please use a fix grid layout to prevent lower page margin, If required limit motion title to _one_ line with "..."

I am not sure what you mean here.

**Comment by emanuelschuetze** [comment]
Sorry, I misunderstand this PR. You changed the motion poll not the ballot pdf ... I retest it...

**Comment by MaximilianKrambach** [comment]
This was a prematurely closed condition resulting in impossible values. Fixed

File: client/src/app/site/motions/services/motion-pdf.service.ts [hunk 1]
```diff
@@ -243,6 +247,62 @@ export class MotionPdfService {
         }
 
         // TODO: Voting result, depends on polls
```
Comment (tsiegleauq) [inline client/src/app/site/motions/services/motion-pdf.service.ts]: comment can be removed

**Pull request merged**